lib/pages/player/player_item_panel.dart

```diff
@@ -701,8 +701,8 @@ class _PlayerItemPanelState extends State<PlayerItemPanel> {
           // 自定义播放器底部组件
           Positioned(
             bottom: 0,
-            left: 0,
-            right: 0,
+            left: MediaQuery.of(context).padding.left,
+            right: MediaQuery.of(context).padding.right,
```

Comment:
```suggestion
            left: 0,
            right: 0,
```

这里底下有现成的 SafeArea，直接修改 left 和 right 就可以